.idea/dictionaries/ndkoval.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="ndkoval">
```

Comment:
Please, give it some neutral name (like `shared`).

Reply:
In IDEA each user has its own dictionary to reduce merge conflicts with the global dictionary, but all these per-user dictionaries are used by IDEA independently on the current user.

Reply:
That's exactly my point. The dictionary that is committed to the report should not be your personal dictionary that is tied to your user-name, it shall be a common dictionary that is shared across the project.

Reply:
That is not how IntelliJ IDEA works (yes, the behavior is counterintuitive). You cannot create a dictionary with a custom name -- it will be automatically cleaned up by IntelliJ IDEA. Instead, there are per-user dictionaries which can be updated by each user personally -- when you use IntelliJ IDEA **all** these per-user dictionaries are used, so that your IDEA will know about the words listed in `ndkoval` dictionary. The original motivation (provided by the IntelliJ IDEA team) is that in this case you will not have merge conflicts on the dictionary since it is split by users. I, personally, would not agree with this solution (at least, I spent an hour trying to understand the behavior), but that is how it works.

Reply:
In conclusion,
1) you cannot create dictionaries with custom names, and, in particular, a single global dictionary
2) all per-user dictionaries are used in IDEA, independently of the current user

Reply:
Of course, you can. I've committed a fix.